test/core/number/cos.wren

```diff
@@ -0,0 +1,6 @@
+IO.print(0.cos)             // expect: 1
+IO.print(Num.pi.cos)        // expect: -1
+IO.print((2*Num.pi).cos)    // expect: 1
```

Comment:
Are you really sure we can't rely on precise results? I would expect the standard functions on double to be pretty strictly defined, though if not we should of course do a fuzzy compare.

But maybe it can wait until there is actually a test framework? I find this `IO.print` line harder to reach in case of manual testing, and it makes it much less obvious what the problem might be if this test fails.